app/lib/actions/katello/content_view_version/import.rb

```diff
@@ -0,0 +1,34 @@
+module Actions
+  module Katello
+    module ContentViewVersion
+      class Import < Actions::EntryAction
+        PULP_USER = 'pulp'.freeze
+
+        def plan(content_view, path:)
+          content_view.check_ready_to_import!
+          unless SmartProxy.pulp_primary.pulp3_repository_type_support?(::Katello::Repository::YUM_TYPE)
+            fail HttpErrors::BadRequest, _("This API endpoint is only valid for Pulp 3 repositories.")
+          end
+          ::Katello::Pulp3::ContentViewVersion::Import.check_permissions!(path)
+          metadata = ::Katello::Pulp3::ContentViewVersion::Import.metadata(path)
+          major = metadata[:content_view_version][:major]
+          minor = metadata[:content_view_version][:minor]
+
+          if ::Katello::ContentViewVersion.where(major: major, minor: minor, content_view: content_view).exists?
+            fail _("Content View Version specified in the metadata - '%s %s.%s' has already been imported." % [content_view.name, major, minor])
```

Comment:
It would be helpful if this error could tell the user how to address the problem.  I got this and ended up just manually changing the `major` value in my `metadata.json` and it worked fine.

If a user tries an import and gets this error, it's likely it's because the import failed in some way and they are trying again.  What is the recommended course of action then?

Reply:
> If a user tries an import and gets this error, it's likely it's because the import failed in some way and they are trying again. What is the recommended course of action then?

If a user was faced with this, it means it was already imported before. I could add something along "delete the content view version if you want to reimport". Don't mess with the json.

Reply:
I did try deleting the version but got 'Could not find content_view resource with id 3' which is weird, because it's right there in the UI.  Figured that was an unrelated bug.